UnityProject/Assets/Scripts/Chat/Chat.Process.cs

```diff
@@ -96,15 +96,21 @@ private static (string, ChatModifier) ProcessMessage(ConnectedPlayer sentByPlaye
 			chatModifiers |= ChatModifier.Whisper;
 		}
 		// Sing
-		else if (message.StartsWith("%"))
+		else if (sentByPlayer.Script.mind != null &&
+				sentByPlayer.Script.mind.occupation != null &&
+				message.StartsWith("%"))
 		{
 			message = message.Substring(1);
+			message = Sing(message);
 			chatModifiers |= ChatModifier.Sing;
 		}
 		// Sing alias
-		else if (message.StartsWith("/s " ))
+		else if (sentByPlayer.Script.mind != null &&
+				sentByPlayer.Script.mind.occupation != null &&
+				message.StartsWith("/s "))
```

Comment:
These checks were needed for the clown because we look up the player's occupation - but we don't do that for singing, so there's no need to check the player's mind.

Reply:
done